packages/puppeteer-core/src/common/Frame.ts

```diff
@@ -353,14 +353,11 @@ export class Frame {
           referrerPolicy,
         });
         ensureNewDocumentNavigation = !!response.loaderId;
-        return response.errorText
```

Comment:
isn't it used on line 313?

Reply:
Yes yes. Sorry about that. It was late in the night :p